app/src/release/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt

```diff
@@ -0,0 +1,10 @@
+package co.netguru.baby.monitor.client.feature.debug
+
+import io.reactivex.Observable
+import javax.inject.Inject
+
+class DebugModule @Inject constructor() {
+    fun sendCryingProbabilityEvent(cryingProbability: Float) {}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/a089e65afe6044fe8b2c4847ffd33998441cf079/app/src/release/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt#L7"></span>Function parameter cryingProbability is unused.</td>
    </tr>
  </tbody>
</table>